pkg/operator/cluster/pool.go

```diff
@@ -0,0 +1,122 @@
+package cluster
+
+import (
+	"fmt"
+
+	rookclient "github.com/rook/rook/pkg/rook/client"
+	"k8s.io/client-go/pkg/api/v1"
+
+	"github.com/rook/rook/pkg/model"
+)
+
+const (
+	replicatedType  = "replicated"
+	erasureCodeType = "reasure-coded"
```

Comment:
doh, how did this work? maybe under the covers it only checks for replicated or !replicated?